models/issue_dependency.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"code.gitea.io/gitea/modules/setting"
+	"time"
+)
+
+// IssueDependency represents an issue dependency
+type IssueDependency struct {
+	ID           int64     `xorm:"pk autoincr"`
+	UserID       int64     `xorm:"NOT NULL"`
+	IssueID      int64     `xorm:"UNIQUE(issue_dependency) NOT NULL"`
+	DependencyID int64     `xorm:"UNIQUE(issue_dependency) NOT NULL"`
+	Created      time.Time `xorm:"-"`
+	CreatedUnix  int64     `xorm:"INDEX created"`
```

Comment:
Please use util.TimeStamp instead of int64 and time.Time.

Reply:
You can find many example on other object.

Reply:
Fixed.

Reply:
Hi, maybe you didn't push success? I cannot find the change.

Reply:
I did push however github didn't got it...
But you can see the change when directly viewing it in my repository: https://github.com/kolaente/gitea/blob/master/models/issue_dependency.go